app/views/thredded/topics/_following_users.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if Thredded.show_following_users_in_topic %>
+  <div class="thredded--topic-following-users">
+    <% if topic.following_users.present? %>
+      Following:
+      <% topic.following_users.each do |user| %>
+        <%= "@#{user}" %>
```

Comment:
@glebm My thinking is that it helps new users to understand how the mentioning system works. And yes, gotcha on the quotes.